src/storage/db_interface_frontend.py

```diff
@@ -203,20 +203,20 @@ def get_latest_comments(self, limit=10):
 
     # --- file tree
 
-    def generate_file_tree_nodes_for_uid_list(self, uid_list, root_uid, whitelist=None):
+    def generate_file_tree_nodes_for_uid_list(self, uid_list: List[str], root_uid: str, parent_uid, whitelist=None):
```

Comment:
Hm. Looks inconsistent to add type hints to some, but not to all parameters (especially the new one)